move citation to home page

superstar54 opened the pull request:
Implemented the suggestions from Giovanni,
- [x] The citation should be much easier to find. I suggest to put it somewhere at the bottom of the homepage (e.g. below Carlo's statement)
- [x] move the citation away from the deployments
- [ ] put citation at the top of the `About` page, before the team.
- [x] Also for now I would replace the sentence "You can [run AiiDAlab yourself](https://aiidalab.readthedocs.io/en/latest/usage/index.html#aiidalab-launch), however we also maintain a few official cloud-backed deployments." with "You can [run AiiDAlab yourself](https://aiidalab.readthedocs.io/en/latest/usage/index.html#aiidalab-launch), however we also maintain a few official cloud-backed deployments for researchers affiliated with some of the projects funding the development of AiiDAlab."

Since the citation is already on the homepage, I think it is not necessary to put it on the `About` page. I feel that it doesn't fit there too. Besides, using the current template, on the `About` page, the title `Our team` is always at the top of the page. It is not easy to put the citation before it. So, I didn't implement this one.

giovannipizzi commented:
Thanks!

Just a few comments:
- I would try to center better the citation in the white part (now it's very close to the grey border up, see image below). I would increase the white space above and reduce below so the whole paragraph more centred).
<img width="800" alt="Screenshot 2023-02-15 at 10 55 36" src="https://user-images.githubusercontent.com/16132028/218994783-fa02d1d8-3dd3-4a86-ae0c-07d00c231bc9.png">

- for some reason, in ABOUT the advisory committee is empty - not sure if it's just an issue of my own local deployment?

- To be on the safe side, I would also add the very same text ("Citation"/"We kindly ask you..." also at the bottom of the about page, below "Our sponsors") (I agree with not putting at the top, but I think people might not see it at the bottom of the home page and go directly to about, so good to duplicate).

- (maybe for a different PR - in this case at least open an issue) is it possible to avoid to link directly to the email of people in the about/team page, but link to their institutional page (where anyway the email is)? This is I think better (e.g. to see their role in the institution etc.) and also avoids to have clean-text emails that increase significantly the spam.

- in the deployment page, the first FAQ should be hidden now (the one about the EGI account) since the public demo is not online anymore 

Then it can be merged!

superstar54 commented:
Thanks! @giovannipizzi . I have addressed all the comments. 
- I increased the white space above. Now it centers better. The space below is set for all sections and can not be modified.
- The error of `advisory committee` is fixed.
- The citation is also added on the `About` page.
- Use the link to the members' institutional page instead of email. For the former contributors, since we don't know the page for all of them, the links are removed.
- The first FAQ is removed.

unkcpz commented:
Hi @superstar54, could you have a look and see if anything need to further settled for this PR? 

And don't forget to add photos of new members to the page.
